<launch>
    <node name="pointcloud_to_pcd0" pkg="pcl_ros" type="pointcloud_to_pcd" output="screen" >
        <remap from="input" to="/k4a_0/depth_to_rgb/filtered_points"/>
        <param name="prefix" value="0/"/>
        <param name="fixed_frame" value="panda_link0" />
    </node>

    <node name="pointcloud_to_pcd1" pkg="pcl_ros" type="pointcloud_to_pcd" output="screen">
        <remap from="input" to="/k4a_1/depth_to_rgb/filtered_points"/>
        <param name="prefix" value="1/"/>
        <param name="fixed_frame" value="panda_link0" />
    </node>

    <node name="pointcloud_to_pcd2" pkg="pcl_ros" type="pointcloud_to_pcd" output="screen">
        <remap from="input" to="/k4a_2/depth_to_rgb/filtered_points"/>
        <param name="prefix" value="2/"/>
        <param name="fixed_frame" value="panda_link0" />
    </node>

    <node name="pointcloud_to_pcd3" pkg="pcl_ros" type="pointcloud_to_pcd" output="screen">
        <remap from="input" to="/k4a_3/depth_to_rgb/filtered_points"/>
        <param name="prefix" value="3/"/>
        <param name="fixed_frame" value="panda_link0" />
    </node>

</launch>